runner/server/handle_secrets.go

```diff
@@ -223,9 +259,12 @@ func (s *Server) handleDeleteSecretKey() http.HandlerFunc {
 		// setSecretKey
 		if err := s.driver.DeleteKey(projectID, name, key); err != nil {
 			logrus.Errorf("Failed to list secret - %s", err.Error())
+			w.Header().Set("Content-Type", "application/json")
```

Comment:
Instead of setting error here. Set it inside `utils.SendErrorResponse`. Do this everywhere